subt/zmq-subt-x2.json

```diff
@@ -11,8 +11,12 @@
             "symmetric": false,
             "virtual_bumper_sec": 60,
             "virtual_bumper_radius": 10.0,
-            "virtual_world": true
-          }
+            "virtual_world": true,
+            "follow_wall": {
+              "max_wall_distance": 7.8,
```

Comment:
This can be 7.6 and Freyja still does not fall from stairs in Urban. It can help her not to return to early from down slopes in Tunnel.

Reply:
do you propose this change also for R2?

Reply:
I am changing both robots and creating build `ver103fall5`

Reply:
`eaff6ff8-bbde-4720-ac30-e3c7411e7e6d` - running Freyja and R2 together, just for fun :)

Reply:
the racing is actually really fun:
![filans-p1-racing](https://user-images.githubusercontent.com/5664353/114387967-60b67480-9b93-11eb-835f-a3ee225c83c2.jpg)